projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -202,6 +203,8 @@ protected function __construct() {
 			)
 		);
 
+		add_action( 'admin_menu', array( $this, 'rename_feedback_menu' ) );
```

Comment:
Hey 👋 , I've tested your changes on my local environment and it seems that the `Export CSV` submenu link is broken. One way to fix this, I think, is to call the method directly here. Not sure though what are the implications of this. 

Also (and unrelated to your changes), I looks like that `Export CSV` page displays a blank page.